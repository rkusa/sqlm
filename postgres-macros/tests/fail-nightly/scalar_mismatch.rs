../fail-stable/scalar_mismatch.rs