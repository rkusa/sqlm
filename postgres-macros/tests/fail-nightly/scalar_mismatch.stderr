error[E0277]: the trait bound `String: Query<Literal<i64>>` is not satisfied
 --> tests/fail-nightly/scalar_mismatch.rs:5:56
  |
5 |     let _: String = sql!("SELECT COUNT(*) FROM users").await.unwrap();
  |                                                       -^^^^^ the trait `Query<Literal<i64>>` is not implemented for `String`
  |                                                       |
  |                                                       help: remove the `.await`
  |
  = help: the trait `Query<Literal<String>>` is implemented for `String`
  = note: required for `Sql<'_, Literal<i64>, String>` to implement `IntoFuture`
